models/user.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, Sequelize) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      this.belongsToMany(models.Designation, {
+        through: models.UserDesignationMapping,
+        foreignKey: 'userId',
+      });
+
+      this.belongsToMany(models.Role, {
+        through: models.UserRoleMapping,
+        foreignKey: 'userId',
+      });
+
+      this.belongsToMany(models.Workspace, {
+        through: models.UserWorkspaceMapping,
+        foreignKey: 'userId',
+      });
+
+      this.belongsToMany(models.Sprint, {
+        foreignKey: 'userId',
+        targetKey: 'id'
+      });
+
```

Comment:
User should be belongToMany with Workspace, Sprint should be belong to a WorkSpace

Reply:
ok